src/Simulation/Simulators.Tests/StackTraceTests.cs

```diff
@@ -0,0 +1,314 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Could we please add the same tests for the TraceSimulator and the QuantumSimulator as well?

Reply:
In this tests, we just need an instance of a class that inherits from `SimulatorBase`, so there is no need to test several simulators.